File: WebApi/Models/Loan/MonthlyInstallmentDataVM.cs
namespace WebApi.Models.Loan
{
    public class MonthlyInstallmentDataVM
    {
        public int Id { get; set; }
        public decimal AmountOfCapitalAndInterest { get; set; }
        public decimal AmountOfInterest { get; set; }
        public decimal AmountOfCapital { get; set; }
        public decimal AmountOfCapitalPaid { get; set; }
        public decimal AmountOfInterestPaid { get; set; }
        public decimal AmountofPrincipalOutstanding  { get; set; }
    }
}

File: WebApi/angular/src/app/shared/model/MonthlyInstallmentDataVM.ts

export interface IMonthlyInstallmentDataVM {
    Id: number;
    AmountOfCapitalAndInterest: number;
    AmountOfInterest: number;
    AmountOfCapital: number;
    AmountOfCapitalPaid: number;
    AmountOfInterestPaid: number;
    AmountofPrincipalOutstanding: number;
}
export class MonthlyInstallmentDataVM implements IMonthlyInstallmentDataVM {
    Id: number;
    AmountOfCapitalAndInterest: number;
    AmountOfInterest: number;
    AmountOfCapital: number;
    AmountOfCapitalPaid: number;
    AmountOfInterestPaid: number;
    AmountofPrincipalOutstanding: number;
}
